src/coin/skycoin/models/skyapi_mock_test.go

```diff
@@ -1,108 +0,0 @@
-package skycoin
```

Comment:
Functions in this file have to remain apart of the rest due to their specific purpose of mocking real classes .